rasa/cli/test.py

```diff
@@ -202,7 +202,10 @@ async def run_nlu_test_async(
         config = rasa.cli.utils.get_validated_path(
             config, "config", DEFAULT_CONFIG_PATH
         )
-        perform_nlu_cross_validation(config, nlu_data, output, all_args)
+        config_importer = TrainingDataImporter.load_from_dict(config_path=config)
```

Comment:
can we not use the `test_data_importer` from above?

Reply:
that is not possible as at the time that importer is instantiated it is still unclear whether config is one particular config or multiple for model comparison mode. The TrainingDataImporter, however, needs a single one.